src/main/java/org/javarosa/core/model/TriggerableDag.java

```diff
@@ -398,6 +395,82 @@ public void finalizeTriggerables(FormInstance mainInstance) throws IllegalStateE
         return edges;
     }
 
+    private static Set<QuickTriggerable> getDependantTriggerables(FormInstance mainInstance, EvaluationContext ec, QuickTriggerable triggerable, Map<TreeReference, Set<QuickTriggerable>> triggerIndex) {
```

Comment:
The corresponding methods should be removed from `FormInstanceParser` as they're no longer used, right?